enroll.php

```diff
@@ -0,0 +1,109 @@
+<?php 
+include'head.php';  
+ require_once'bd.php';
+
+if(isset($_POST['submit1'])){
+    
+    $fname=$_POST['fname'];
+    $lname=$_POST['lname'];
+    $user=$_POST['username'];
+    $pass1=$_POST['pass1'];
+    $pass2=$_POST['pass2'];
+    $email=$_POST['email'];
+    $occupation=$_POST['profession'];
+    $date=$_POST['date1'];
+    $tel=$_POST['tel1'];
+    $country=$_POST['pays'];
+    $sexe=$_POST['sex1'];
+    
+    if($pass1!=$pass2){
+        
+        $erreu='Your passwords are differents. Please write the same password';
+    }
+    
+   
+      else{ $quser = $bdd->prepare("SELECT * FROM register WHERE username=?");
+             $quser->execute(array($user));
+              $test = $quser->rowCount();
+        if($test>=1){
+            
+            $erreu='This username is already used. please change another one.';
+           
+        }
+       else{
+       $repon=$bdd->prepare("INSERT INTO register SET fname=?, lname=?, username=?, pass1=?, pass2=?, email=?,occupation=?, date=?,number=?,sex=?,country=?");
+       $pass1 = sha1($_POST['pass1']);
+       $pass2 = sha1($_POST['pass2']);

```

Comment:
You don't have to store the password twice. You only use the second password to make sure the user inserted a password they really wanted to insert (without a type).

Moreover, you should enforce a minimum length for the password (e.g. 8 characters).